### ISSUE TITLE
Briefly, the FAQ page was reverting to previously-set language

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1335]
**TL;DR:** Somehow I got into a state where the FAQ page on Staging or Production had started reverting to the last language I had set it to, rather than letting me change it. 

I was changing it without issue earlier today. I think this must somehow be user error (= me) but I don't see how yet - I am not in an incognito window, haven't disabled cookies, etc. After navigating to the homepage, whose language I could change, I went back to the FAQ page, and was again **able** to change the language there as well.

I don't know how to repro this, now that it stopped happening. When it was happening, it was easy to repro - just try to change the language on the FAQ page.

(Production, Chrome 58.0.3029.110, Mac OS El Capitan)
![out](https://cloud.githubusercontent.com/assets/1320456/26655963/0ecd9a0c-462b-11e7-9be6-e359a99462ed.gif)

**Comment by jwzimmer** [comment]
I just ran in to this again, on Staging.
![out](https://cloud.githubusercontent.com/assets/1320456/26735631/b7d3401e-4790-11e7-9598-2d6667b3a150.gif)

Does this correspond to when something is happening with the site, like a deploy/ rollback?

**Comment by jwzimmer** [comment]
I was asking @mewtaylor & @colbygk a couple questions about this - 
Matt was redeploying to Staging on scratch-api at about the time I saw this bug the 2nd time.
Colby said this could be caused by breaking the set-up between fastly and backend, and that part of the deploy does involve updating scratch-www rules in fastly.
However, scratch-api wouldn't seem to involve that, so the timing could be a coincidence?

**Comment by jwzimmer** [comment]
Currently on Staging I'm seeing Explore > Projects > All I'm seeing "All" in Brazilian Portuguese but the other sections (Games, Art) in English. When I try to choose English under "all", the page stays in Brazilian Portuguese.

**Comment by jwzimmer** [comment]
Ran into this again on Production. I tried restarting Chrome to see if that would fix the issue; it did not, but the issue seemed to fix itself a few minutes later.

![out](https://user-images.githubusercontent.com/1320456/26847601-a5ef3c48-4acb-11e7-877f-9ab4608e9493.gif)

**Comment by jwzimmer** [comment]
**Next time I run into this bug, TODO:**
- [ ] Check values of the 2 Scratch language cookies
- [ ] Preserve log & check network tab when trying to change language
- [ ] Check other www pages
- [ ] Check scratchr2 pages

From looking at the cookies now (the bug is NOT currently happening), the values are:
<img width="986" alt="screen shot 2017-06-06 at 3 34 16 pm" src="https://user-images.githubusercontent.com/1320456/26848209-a353fed6-4acd-11e7-99aa-565c5fb73349.png">
<img width="984" alt="screen shot 2017-06-06 at 3 34 06 pm" src="https://user-images.githubusercontent.com/1320456/26848210-a3672416-4acd-11e7-9b25-95493a398227.png">

Note that @rschamp didn't know why the expiration date for the cookie was set to 5/31/18, as it should be 1 year from now, 6/5/17 (so, 6/5/18).

**Comment by jwzimmer** [comment]
Here are the cookie values and what language the page is in:

Set 1
<img width="782" alt="screen shot 2017-06-06 at 4 21 03 pm" src="https://user-images.githubusercontent.com/1320456/26850040-2d85dd76-4ad4-11e7-8339-68874f6e33f3.png">
<img width="973" alt="screen shot 2017-06-06 at 4 20 56 pm" src="https://user-images.githubusercontent.com/1320456/26850041-2d8b8cd0-4ad4-11e7-8b52-92be1ab2f48e.png">

Set 2
<img width="778" alt="screen shot 2017-06-06 at 4 20 37 pm" src="https://user-images.githubusercontent.com/1320456/26850043-2d98cf08-4ad4-11e7-8a20-c991174c679b.png">
<img width="986" alt="screen shot 2017-06-06 at 4 20 26 pm" src="https://user-images.githubusercontent.com/1320456/26850042-2d91fe08-4ad4-11e7-8495-9d683da6bce9.png">

Unfortunately the bug stopped occurring before I could try other pages. 

I do have the network log:
<img width="1280" alt="screen shot 2017-06-06 at 4 24 26 pm" src="https://user-images.githubusercontent.com/1320456/26850190-ada76164-4ad4-11e7-8412-2bc4539ce551.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 24 20 pm" src="https://user-images.githubusercontent.com/1320456/26850191-ada78a9a-4ad4-11e7-939c-2ec1786283b2.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 24 08 pm" src="https://user-images.githubusercontent.com/1320456/26850192-adac6fce-4ad4-11e7-8fc3-2c34f1307e21.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 24 00 pm" src="https://user-images.githubusercontent.com/1320456/26850188-ada6770e-4ad4-11e7-8c14-831f8d223d59.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 23 48 pm" src="https://user-images.githubusercontent.com/1320456/26850189-ada66674-4ad4-11e7-9ef0-a786d2c5c6d7.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 23 40 pm" src="https://user-images.githubusercontent.com/1320456/26850196-adb635d6-4ad4-11e7-9dfe-eb9d187926fa.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 23 31 pm" src="https://user-images.githubusercontent.com/1320456/26850193-adafce12-4ad4-11e7-8cc0-25db59cf98d1.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 23 23 pm" src="https://user-images.githubusercontent.com/1320456/26850195-adb00436-4ad4-11e7-94b6-9daa24b6c089.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 23 14 pm" src="https://user-images.githubusercontent.com/1320456/26850194-adb01f98-4ad4-11e7-90cf-6ad2707ada36.png">
<img width="1280" alt="screen shot 2017-06-06 at 4 22 59 pm" src="https://user-images.githubusercontent.com/1320456/26850197-adbfaf8a-4ad4-11e7-9b72-35097d192e52.png">
